src/viur_cli/cloud.py

```diff
@@ -16,12 +16,48 @@ def cloud():
 
 
 @cloud.command(context_settings={"ignore_unknown_options": True})
-@click.argument("action", type=click.Choice(["backup"]))
-def enable(action):
+@click.argument("action", type=click.Choice(["backup","api"]))
+@click.argument("profile", default="default")
+def enable(action, profile):
     """Enable a specific action based on the provided parameter."""
     if action == "backup":
         enable_gcp_backup()
 
+    if action == "api":
+        enable_api(profile)
+
+
+def enable_api(profile):
+    """Enables a Google Cloud Service/API.
+
+    Args:
+        project_id (str, optional): The Google Cloud project ID. Defaults to None.
+
+    Raises:
+        click.ClickException: If an error occurs during project setup or service enablement.
+    """
+    project_id = config.get_profile(profile)["application_name"]
+    if not project_id:
+        project_id = click.prompt("Please enter your Google Cloud project ID")
+
+    project_setup_command = f"gcloud config set project {project_id}"
```

Comment:
This is an unwanted side affect. Don't change the project without changing it back. However just use the `--project` flag.